c2corg_ui/static/js/lang.js

```diff
@@ -70,7 +74,7 @@ app.LangController = function(gettextCatalog, langUrlTemplate,
    */
   this.lang = this.cookies_.get('interface_lang') ||
       ngeoGetBrowserLanguage(this['langs']) || 'fr';
-  this.updateLang();
+  this.updateLang(null);
```

Comment:
changed to this.updateLang(this.lang);
and lang = lang || null;  

and it works well :)) thx